goal_src/jak1/pc/progress-pc.gc

```diff
@@ -1182,6 +1208,209 @@
                   (reset-original-camera *pc-settings*)
                   (sound-play "cursor-options")
                   )
+                 ;; LEVEL SELECT
+                 (((game-text-id training-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; extra setup for IL speedruns
+                      (speedrun-set-common-settings)
+                      ))
+                    )
+                  (initialize! *game-info* 'game (the-as game-save #f) "game-start")
+                  )
+                 (((game-text-id village1-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; give enough orbs to buy all cells 120+120+90+90=420
+                      (set! (-> *game-info* money-total) 420.0)
+                      (set! (-> *game-info* money) 420.0)
+                      ;; extra setup for IL speedruns
+                      (speedrun-set-common-settings)
+                      ))
+                    )
+                  (initialize! *game-info* 'game (the-as game-save #f) "village1-hut")
```

Comment:
For ILs Maybe village1-warp should be considered

Reply:
will fix this, blue eco in FC, and klaww cutscene tonight